### DISCUSSION TITLE
How to update PHP on macOS 10.13.6 Server with brew

### DISCUSSION THREAD

**Discussion by bbonev80** [Homebrew/discussions#2740] [Tap maintenance and Homebrew development]
Guy,

I need to update the supplied 7.1.33 with macOS server to lets say PHP 8.1.1 or any 8 version. Can somebody write a guide or maybe create a brew formula for this? Keep in mind this is not the regular php install since the PHP used by server stays inside Library/Server/Web/

Much appreciated.

**Comment by SMillerDev** [comment]
Homebrew can't write in another location and homebrew php can't work with the build-in server.

**Comment by bbonev80** [comment]
I see. I used macports to update CLAMAV which had server specific formula, so I thought to ask.
